InscryptionAPI/Card/ExtendedAbilityBehaviour.cs

```diff
@@ -89,22 +63,24 @@ private static List<ExtendedAbilityBehaviour> GetHealthBuffs(PlayableCard card)
     [HarmonyPostfix]
     private static void AddPassiveAttackBuffs(ref PlayableCard __instance, ref int __result)
     {
-        if (__instance.slot == null)
-            return;
+        if (TurnManager.Instance.GameEnding) return;
```

Comment:
`__instance.slot` can still be not null while the game is ending. For example, I added some logging in my local to determine how to fix this and this is what I got.

```csharp
[Debug  :GrimoraMod] [GModBattleSequencer] Adding [Skeleton] to cardsThatHaveDiedThisGame. Current count [1]
[Debug  :InscryptionAPI] [GetPassiveAttackBuffs] Not getting buffs as the game is ending.
[Debug  :InscryptionAPI] [GetPassiveAttackBuffs] Not getting buffs as the game is ending.
[Debug  :InscryptionAPI] [GetPassiveAttackBuffs] Not getting buffs as the game is ending.
[Debug  :InscryptionAPI] [GetPassiveAttackBuffs] Not getting buffs as the game is ending.
[Debug  :InscryptionAPI] [GetPassiveAttackBuffs] Not getting buffs as the game is ending.
[Debug  :GrimoraMod] [UpdateAbilityIcons] Attempting to setup new positions/scaling for card [Skeleton (DiskCardGame.CardInfo)]
[Debug  :GrimoraMod] [UpdateAbilityIcons] Attempting to setup new positions/scaling for card [Skeleton (DiskCardGame.CardInfo)]
[Debug  :InscryptionAPI] [GetPassiveAttackBuffs] Not getting buffs as the game is ending.
[Debug  :InscryptionAPI] [GetPassiveAttackBuffs] Not getting buffs as the game is ending.
[Debug  :GrimoraMod] [UpdateAbilityIcons] Attempting to setup new positions/scaling for card [Skeleton (DiskCardGame.CardInfo)]
[Debug  :GrimoraMod] [UpdateAbilityIcons] Attempting to setup new positions/scaling for card [Skeleton (DiskCardGame.CardInfo)]
```